projects/plugins/jetpack/modules/theme-tools/random-redirect.php

```diff
@@ -11,7 +11,7 @@
  * @package automattic/jetpack
  */
 
-// phpcs:disable WordPress.Security.NonceVerification -- No changes to the site here, it just redirects.
+// phpcs:disable WordPress.Security.NonceVerification, WordPress.Security.ValidatedSanitizedInput.InputNotValidated -- No changes to the site here, it just redirects.
```

Comment:
A small update to appease the wpcom linting.

Reply:
It would probably be better to actually fix these.

* Line 49 I made a suggestion inline.
* Line 59 is safe to individually ignore, it's just testing for equality. Or you could do `filter_var( wp_unslash( $_SERVER['HTTP_USER_AGENT'] ) )`, which is kind of dumb but works.
* Lines 69 and 70 want to be run through `sanitize_key()`, which will handle both errors on each line.

Reply:
Thanks, I removed the `InputNotValidated` ignore, and L49 is fixed up. With the `NonceVerification` ignore still in use, I'm not seeing any lint errors here or on wpcom, should I be seeing some?

Reply:
Oh, I guess wpcom isn't seeing lines 59, 69, or 70 since it uses phpcs-changed.